src/libs/actions/PaymentMethods.js

```diff
@@ -311,6 +312,42 @@ function dismissSuccessfulTransferBalancePage() {
     Navigation.navigate(ROUTES.SETTINGS_PAYMENTS);
 }
 
+/**
+ * Looks through each payment method to see if there is an existing error
+ * @param {Object} bankList
+ * @param {Object} cardList
+ * @returns {Boolean}
+ */
+function hasPaymentMethodError(bankList, cardList) {
+    const combinedPaymentMethods = lodashMerge(bankList, cardList);
+    return _.some(combinedPaymentMethods, item => !_.isEmpty(item.errors));
```

Comment:
Why do we check the cardList if we didn't add the `errors` and `pendingAction` keys [here](https://github.com/Expensify/App/blob/2debc84873eb3b463c9b34e2ec81ad2f83e50d66/src/libs/PaymentUtils.js#L66-L82) like you did with the bank account items? Is that intentional?

From the looks of the code, it looks like it was written generic enough to handle bank accounts and cards... so maybe you missed adding those keys in PaymentUtils/cards?

Reply:
Good catch, I just missed adding those keys for the cards